packages/reakit/src/Toolbar/__examples__/ToolbarWithMenu/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { RefObject } from "react";
+import { useToolbarState, Toolbar, ToolbarItem } from "reakit/Toolbar";
+import { useMenuState, MenuButton, Menu, MenuItem } from "reakit/Menu";
+import { Button } from "reakit/Button";
+import { ToolbarProps } from "../../Toolbar";
+
+const MoreItems = React.forwardRef<HTMLButtonElement, ToolbarProps>(
+  (props, ref) => {
+    const menu = useMenuState({ placement: "bottom-end" });
+    const buttonRef = ref;
+
+    return (
+      <>
+        <MenuButton
+          {...menu}
+          {...props}
+          ref={buttonRef}
```

Comment:
We can just pass the ref directly.
```suggestion
    return (
      <>
        <MenuButton
          {...menu}
          {...props}
          ref={ref}
```